tests/integration/test_select.py

```diff
@@ -196,22 +194,31 @@ def test_timestamp_casting(c, input_table, request):
         """
     )
 
-    expected_df = datetime_table.astype("<M8[ns]")
+    expected_df = datetime_table
+    expected_df["timezone"] = (
+        expected_df["timezone"].astype("<M8[ns]").dt.floor("D").astype("<M8[ns]")
+    )
+    expected_df["no_timezone"] = (
+        expected_df["no_timezone"].astype("<M8[ns]").dt.floor("D").astype("<M8[ns]")
+    )
+    expected_df["utc_timezone"] = (
+        expected_df["utc_timezone"].astype("<M8[ns]").dt.floor("D").astype("<M8[ns]")
+    )
+
     assert_eq(result_df, expected_df)
 
 
-@pytest.mark.skip(reason="WIP DataFusion")
 def test_multi_case_when(c):
     df = pd.DataFrame({"a": [1, 6, 7, 8, 9]})
     c.create_table("df", df)
 
     actual_df = c.sql(
         """
     SELECT
-        CASE WHEN a BETWEEN 6 AND 8 THEN 1 ELSE 0 END AS C
+        CASE WHEN a BETWEEN 6 AND 8 THEN 1 ELSE 0 END AS "C"
```

Comment:
does datafusion not handle this case where the renamed column is not in `"`

Reply:
No, and in fact it seems like we were doing this wrong all along and that in order for case to be honored, at least by default, on postgres the name must be in `"`. Otherwise the name is always converted to lowercase in postgres.